pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -244,6 +247,9 @@
     public static final String CONFIG_OF_QUERY_EXECUTOR_TIMEOUT = "pinot.server.query.executor.timeout";
     public static final String CONFIG_OF_QUERY_EXECUTOR_CLASS = "pinot.server.query.executor.class";
     public static final String CONFIG_OF_REQUEST_HANDLER_FACTORY_CLASS = "pinot.server.requestHandlerFactory.class";
+    public static final String CONFIG_OF_SERVER_SLOW_QUERY_LATENCY_THRESHOLD_MS =
+        "pinot.server.slow.query.latency.thresholdMs";
+    public static final long DEFAULT_SERVER_SLOW_QUERY_LATENCY_THRESHOLD_MS = 100L;
```

Comment:
It looks that we are re-using the existing default threshold but it's pretty surprising that we set this low number for logging slow query and not have seen too much of the logs.

Reply:
Maybe we should change the default value to a larger number `200ms` or `300ms` perhaps. I checked my production use case, some server can have log file size > 20G.